book/src/intro.md

```diff
@@ -60,7 +60,8 @@ weigh before using them, especially in regards to performance and scalability.
 [ud]: https://www.unrealengine.com/
 [jme]: http://jmonkeyengine.org/
 
-We think that basing the Amethyst engine on good and modern principles will allow us to make an open source game engine that can actually be more performant than those engines.
+We think that basing the Amethyst engine on good and modern principles will allow us to make an open source game engine
+that can actually be more performant than those engines.
```

Comment:
Not super helpful and kind of... arrogant sounding?  I'd change it to


We believe that through utilization of a cache friendly ECS we can develop an open source engine that will go toe to toe with these commercial offerings.

Reply:
Don't we have a similar sentence in the Readme by Eyal that sounds pretty similar? I don't think this sounds arrogant, but.. I don't care.

Reply:
Yeah sorry that sentence was garbage..

Reply:
Ah yeah, I never wrote that sentence btw, I just inserted a newline.